lib/apachetest.pm

```diff
@@ -52,6 +52,7 @@ sub setup_apache2 {
 
     if ($mode eq "PHP7") {
         push @packages, qw(apache2-mod_php7 php7);
+        push @packages, qw(php7-cli) if is_tumbleweed;
```

Comment:
Generally, the expectiation is 'work by default on TW - 'workaround for other distros'

so this should rather be something along the lines "add php7-cli unless SLE or Leap"

Reply:
I thought about that as well, but `if is_tumbleweed` seems better to me than `unless (is_sle || is_leap || is_microos('suse')`, especially because the latter is currently unversioned

Reply:
Changed, VR still green

Reply:
Perfect